castervoice/lib/merge/state/stackitems.py

```diff
@@ -63,10 +63,13 @@ def put_time_action(self):
         if settings.SETTINGS["miscellaneous"]["print_rdescripts"] and self.show:
             # formats rdescript with the given data
             try:
-                rd = self.rdescript % self.dragonfly_data
-            except:
+                rd = self.rdescript % self.dragonfly_data if self.dragonfly_data else self.rdescript
```

Comment:
The bug that this addresses I could not reproduce with the original settings window but I did have it occur with your QT PR. Hopefully this resolves an issue